yang/goldstone-static-route.yang

```diff
@@ -0,0 +1,127 @@
+module goldstone-static-route {
+
+  yang-version "1";
+
+  namespace "http://goldstone.net/yang/goldstone-static-route";
+  prefix gs-static-route;
+
+  import goldstone-routing {
+    prefix gs-rt;
+  }
+
+  import goldstone-nexthop {
+    prefix gs-nexthop;
+  }
+
+  import ietf-inet-types {
+    prefix inet;
+  }
+
+  organization
+    "GoldStone";
+
+  description
+    "This module contains a collection of YANG definitions for
+    managing and configuring static route";
+
+  revision 2022-12-14 {
+    description
+      "Initial revision.";
+  }
+
+  grouping path-list-config {
```

Comment:
How do we use `table-id` and `tag`?
Can OcNOS and `south-ocnos` support these leaves?

Reply:
Tags can be used for filtering and route redistribution for other protocols. It is supported by OcNOS
OcNOS does not support table-id. That seems to be a concept tied to the Linux Routing subsystem and a way to implement VRFs using kernel vrf interfaces. In my opinion, table-id is VRF implementation dependent and should not be exposed to the final user

Reply:
Thank you so much for your comments.
I defined them for the same reason with https://github.com/oopt-goldstone/goldstone-mgmt/pull/73#discussion_r1053918121.
Since our first target is static route support, `table-id` and `tag` may not be needed in this phase.

Reply:
I agree. Let's remove both of them. We can add `tag` when we support route redistribution.